### ISSUE TITLE
A11y_SQLdatabase projects extension_Change Target Platform_Voiceover : Voiceover is announcing as "Non-selected" for currently selected "Target Platform".

### ISSUE THREAD

**Issue by manaswita-chougule** [microsoft/azuredatastudio#22186]
### GitHub Tags:
#A11yMAS; #A11ySev3; #A11yTCS; #DesktopApp; #SH-SQL Azure Data Studio-MAC-March23; #SQL Azure Data Studio; #MAC; #AINotInScope; #MAS1.3.1; #Info and Relationships; #Element:listitem; #Voiceover;

### Actual Result:​
Voiceover is correctly announcing the place holder text for "Change target Platform" as "SQL 2016 is currently selected" but while navigating through list when voiceover focus lands on "SQL2016" it is announcing as "Non-selected".
### Expected Result:​
Voiceover should announce the currently selected target platform as selected in Change Target Platform list items, the same information should be conveyed as placeholder text about current platform there should not be any mismatch.

### Environment Details:​
Application Name: SQL Azure Data Studio
Version: 1.42.0-insider (Universal)
MAC Version: macOS Ventura 13.3

### Repro Steps:

1.Start Voiceover with using shortcut key Command+F5.
2.Launch Azure Data Studio
3.Tab till extension and search for “SQL Database Projects” and install it.
4.Tab till database project and press enter.
5.“Database Project” panel will open.
6.Tab till existing project name and press context menu button(right click).
7.Tab till "Change Target Platform" menu-item in list and press enter.
8.Select box for changing target platform will appear having list of other target platforms.
9.Tab till selected platform name in list.
10.Verify whether Voiceover is announcing as "Non-selected" for currently selected "Target Platform" or not.

### User Impact:
Voiceover user will get confuse if for selected target platform in placeholder value it is announcing as current platform but in list item it is announcing as non-selected.

### Attachment:

![change target platform - announcing as non selected for selected sql](https://user-images.githubusercontent.com/91586101/223456600-fd70e8d4-5c5d-4e9f-acd0-8e1f0d7ab941.gif)
